lib/data/schema.js

```diff
@@ -432,7 +432,11 @@ const compare = (prevFields, fields) => {
   for (let i = 0; i < fields.length; i += 1) {
     if (!(prevFields[i].path === fields[i].path &&
       prevFields[i].order === fields[i].order &&
-      prevFields[i].type === fields[i].type)) {
+      prevFields[i].type === fields[i].type &&
+      // field.selectMultiple may be true, null, or undefined.
+      // null should be seen as equal to undefined here.
+      // eslint-disable-next-line eqeqeq
+      prevFields[i].selectMultiple == fields[i].selectMultiple)) {
```

Comment:
Could also do something like

```js
(prevFields[i].selectMultiple === true) === (fields[i].selectMultiple === true)
```

or

```js
(!!prevFields[i].selectMultiple) === (!!fields[i].selectMultiple)
```

if you also wanted to account for `false`. Though it sounds like we don't currently ever set `selectMultiple` to `false`.

Reply:
(`false == null` is `false`.)